odoo/addons/base/models/ir_actions.py

```diff
@@ -284,6 +285,14 @@ def _existing(self):
         return set(row[0] for row in self._cr.fetchall())
 
 
+    def _get_readable_fields(self):
+        return super()._get_readable_fields() | {
+            "context", "domain", "filter", "filter", "groups_id", "limit",
```

Comment:
`filter` is present twice, also is it even useful?

Reply:
in case one gets remove by the set